src/main/kotlin/net/clgd/ccemux/emulation/tror/CapabilitiesPacket.kt

```diff
@@ -0,0 +1,5 @@
+package net.clgd.ccemux.emulation.tror
+
+class CapabilitiesPacket(val caps: Set<String>) : TRoRPacket<Set<String>>(caps) {
```

Comment:
The tests haven't been ported yet and aren't being run, otherwise they would be failing here. The `toString` methods should be overridden as it was in the xtend version, producing an output like `SP:;-ext1-ext2-ext3-`